2-mandatory/6-journey-planner.js

```diff
@@ -81,7 +84,12 @@ function getTransportModes() {}
 
   Hint: Use the corresponding array method to decide if an element is included in an array.
 */
-function isAccessibleByTransportMode() {}
+function isAccessibleByTransportMode(arr, str) {
+  if(arr.includes(str)){
```

Comment:
here you could just return the result of `.includes()` because it is already a boolean